app/src/main/java/openfoodfacts/github/scrachx/openfood/analytics/AnalyticsEvent.kt

```diff
@@ -26,7 +26,13 @@ sealed class AnalyticsEvent(val category: String, val action: String, val name:
 
     object RobotoffLoggedInAfterPrompt : AnalyticsEvent("user-account", "logged-in-after-prompt", "robotoff", null)
 
-    object ShoppingListCreated : AnalyticsEvent("shopping-lists", "created", null, null)
+    data class ShoppingListCreated(val listName: String?) : AnalyticsEvent("shopping-lists", "created", listName, null)
```

Comment:
Do we really want to know about the product list name? Isn't it something a little bit privacy-concerning?

Reply:
My idea behind thisn was to differentiate what kind of list the user create.
But clearly I can remove it if neccessary. What do you think @teolemon?

Reply:
probably overkill. We're more focused on volumetry.